src/main/java/nl/knaw/dans/filemigration/db/EasyFileDAO.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.filemigration.db;
+
+import io.dropwizard.hibernate.AbstractDAO;
+import nl.knaw.dans.filemigration.api.EasyFile;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.query.Query;
+
+import java.util.List;
+
+public class EasyFileDAO extends AbstractDAO<EasyFile> {
+
+  public EasyFileDAO(SessionFactory sessionFactory) {
+    super(sessionFactory);
+  }
+
+  public List<EasyFile> findByDatasetId(String id) {
+    // TODO createNamedQuery(EasyFile.FIND_BY_DATASET_ID, EasyFile.class)
+    Session session = currentSession();
+    Query<EasyFile> query = session.createSQLQuery("SELECT * FROM easy_files WHERE dataset_sid = ? ORDER BY path");
```

Comment:
Changing this line into 
```suggestion
    Query<EasyFile> query = session.createQuery("SELECT f FROM easy_files f WHERE dataset_sid = ? ORDER BY f.path", EasyFile.class);
ORDER BY path");
```

causes

    org.hibernate.hql.internal.ast.QuerySyntaxException: easy_files is not mapped

Reply:
@jo-pol The HSQL Query seems off to me:
```
"SELECT f FROM nl.knaw.dans.filemigration.api.EasyFile f WHERE dataset_sid = ? ORDER BY f.path"
```
Should be:
```
"SELECT f FROM nl.knaw.dans.filemigration.api.EasyFile f WHERE f.datasetSid = :datasetSid ORDER BY f.datasetSid"
```